lfucg-exactions/server/accounts/serializers.py

```diff
@@ -17,8 +17,8 @@ def to_representation(self, obj):
         return LotSerializer(obj).data
 
 class AccountSerializer(serializers.ModelSerializer):
```

Comment:
I believe rest framework is smart enough to inspect your model fields for `null=True` and `blank=True`.  These get translated -- `Serializer.required=not Model.blank` and `Serializer.allow_null=Model.null`

Adding `null=True` will require a migration, but is probably more explicit and can be enforced by the database.

Reply:
I was getting serializer validation errors from it saying `Lot: This field must not be null`. Within the serializer documentation I found that `required=False` validates the data if the it doesn't exist but we are sending `Lot = None` with the data. The `allow_null=True` allows the validation to pass with `Lot=None` being sent. Does that make sense @chris-allen and am I understanding that correctly?

Reply:
Here's what I was referencing... http://www.django-rest-framework.org/api-guide/fields/

Reply:
You're understanding of those rest framework flags is correct.  I'm just saying that you might get that functionality for free by updating your model to use `null=True` and `blank=True`.

https://stackoverflow.com/questions/8609192/differentiate-null-true-blank-true-in-django#answer-8609425

Reply:
Oh ok- if I'm not mistaken it is that way in the model. 

`lot = models.ForeignKey(Lot, related_name='ledger_lot', blank=True, null=True)`